site/frontend/src/pages/compare/compile/table/benchmark-detail.vue

```diff
@@ -455,40 +433,20 @@ onMounted(() => {
         </div>
       </div>
     </div>
-    <div class="command">
-      <div class="title bold">
-        Local profiling command<Tooltip>
-          Execute this command in a checkout of
-          <a href="https://github.com/rust-lang/rustc-perf">rustc-perf</a>,
-          after a `cargo build --release`, to generate a Cachegrind profile.
-        </Tooltip>
-      </div>
-
-      <select @change="changeProfileCommand">
-        <option
-          :value="ProfileCommand.Diff"
-          :selected="profileCommand === ProfileCommand.Diff"
-        >
-          Diff
-        </option>
-        <option
-          :value="ProfileCommand.Before"
-          :selected="profileCommand === ProfileCommand.Before"
-        >
-          Baseline commit
-        </option>
-        <option
-          :value="ProfileCommand.After"
-          :selected="profileCommand === ProfileCommand.After"
-        >
-          Benchmarked commit
-        </option>
-      </select>
-
-      <CachegrindCmd
-        :commit="profileCommit"
-        :baseline-commit="profileBaselineCommit"
-        :test-case="testCase"
+    <div class="shortcut">
+      <template v-if="props.metric === BINARY_SIZE_METRIC">
```

Comment:
(drive-by note that this check reminded me, would we also want to hide the "frontend/backend/linker" wall-time sections when displaying the size metrics?)

Reply:
The FE/BE/Linker chart doesn't fit that well into the detail, since it's not metric-specific :/ But I wanted it to be easily discoverable. It really only makes sense currently for wall-time, or task-clock.

Maybe we could somehow highlight this fact under its title, like we do for the 30 day charts, where we explain what do they show.

Reply:
I'm finding a bit less helpful than I hoped it would be (as discussed in a recent regression);  we could make a bunch of improvements there. Hiding the chart on the size metrics seems like a straightforward one we probably should do (as it's very big)

Reply:
Ok, I'll send a PR, right after this one gets merged.